drivers/flash/flash_stm32g4x.c

```diff
@@ -20,6 +20,9 @@ LOG_MODULE_REGISTER(LOG_DOMAIN);
 
 #define STM32G4X_PAGE_SHIFT	11
 
+#define STM32G4_FAMILY_MAX_FLASH	512
```

Comment:
Nit: STM32 is the family, G4 is one series. So it would be `STM32G4_SERIES_MAX_FLASH`

Reply:
Done